working/1426-extension-types/feature-specification-views.md

```diff
@@ -319,31 +515,25 @@ but this is ambiguous since
 <code>V<T<sub>1</sub>, .. T<sub>k</sub>>(o)</code>
 can be a view constructor invocation.  Similarly,
 <code>V<T<sub>1</sub>, .. T<sub>k</sub>>.m(o, args)</code>
-is similar to a static method invocation and it may match the semantics
-quite well, but that is also confusing because it looks like actual source
-code, but it couldn't be used in an actual program.*
-
-*Let us compare view methods to extension methods, noting that they are
-similar in many ways. With an extension declaration `E`,
-<code>E<T<sub>1</sub>, .. T<sub>k</sub>>(o).m(args)</code>
-denotes an explicit invocation of the extension member
-named `m` declared by the extension `E`, with `o` bound to `this`, the type
-parameters bound to <code>T<sub>1</sub>, .. T<sub>k</sub></code>,
-and value parameters bound to the values of `args`.  If `V` is a view with
-the same on-type, type parameters, and same declaration of a member `m`,
-<code>invokeViewMethod(V, <T<sub>1</sub>, .. T<sub>k</sub>>, o).m(args)</code>
-denotes an invocation of the view method `m` with the same bindings.*
-
-The static analysis of `invokeViewMethod` is that it takes exactly three
-positional arguments and must be the receiver in a member access. The first
-argument must be a type name that denotes a view declaration, the next
-argument must be a type argument list, together yielding a view type
-_V_. The third argument must be an expression whose static type is _V_ or
-the corresponding instantiated on-type (defined below). The member access
-must be a member of `V` or an associated member of a superview of `V`.
+is similar to a named constructor invocation, but that is also
```

Comment:
I don't really understand the motivation for this whole section.  Why do we need a syntax here?  Why not just specify the semantics?